akita/src/internal/crud.ts

```diff
@@ -46,7 +46,7 @@ export class CRUD {
     };
   }
 
-  _add<S extends EntityState, E>(state: S, entities: E[], idKey): S {
+  _add<S extends EntityState, E>(state: S, entities: E[], idKey, params: AddParams = {}): S {
```

Comment:
you don't need an empty object here as you are passing it from the outside already.

Reply:
what should I put in ? `params?` is necessary because If I dont add it. `crud.spec.ts` test file for `_add()` will show `Expected 4 arguments, but got 3.`

Reply:
make it optional -  `params?`